ChangeLog.md

```diff
@@ -20,6 +20,9 @@ See docs/process.md for more on how version tagging works.
 
 3.1.54 (in development)
 -----------------------
+- Added `--use-port` option to `emcc`. You can use `--show-ports` to get the
```

Comment:
Can you give a little more detail.

How about:

```
Added `--use-port` option to `emcc`.  This option allows ports to be enabled by name
and is designed to replace all the exist `-sUSE_XXX` settings for ports. ....
```

Also, can you end the entry with `(#21214)`?

Reply:
Sure I will. I think it makes sense to emphasize that it will replace the `-sUSE_XXX` syntax eventually. Also I would imagine that no new "official" port (unclear if there is any in the pipe), should add `-sUSE_XXX` and instead use `--use-port` syntax... Certainly contrib ports will.

After seeing the end result, I am quite happy with the new syntax which is in-line with the other `--clear-ports`, `--show-ports` and is essentially a way to add new ports, whether official or contrib in a more generic fashion without adding new settings...

After this last change I will be working on contrib port proper

Reply:
I've been holding off adding any new official ports for several years now, until we had this contrib system in place.